Microsoft.ML.sln

```diff
@@ -135,6 +135,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.SamplesUtils",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.Recommender", "src\Microsoft.ML.Recommender\Microsoft.ML.Recommender.csproj", "{C8E1772B-DFD9-4A4D-830D-6AAB1C668BB3}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.ML.Maml", "Microsoft.ML.Maml", "{29B62884-F67B-456F-AFD1-900802A4032F}"
+	ProjectSection(SolutionItems) = preProject
+		pkg\Microsoft.ML.Maml\Microsoft.ML.Maml.0.8.0-preview-27113-0.nupkg = pkg\Microsoft.ML.Maml\Microsoft.ML.Maml.0.8.0-preview-27113-0.nupkg
```

Comment:
>pkg\Microsoft.ML.Maml\Microsoft.ML.Maml.0.8.0-preview-27113-0.nupkg = pkg\Microsoft.ML.Maml\Microsoft.ML.Maml.0.8.0-preview-27113-0.nupkg [](start = 2, length = 137)

I'm not sure what this line is for. But the fact that it uses exact version string makes me positive that it is incorrect

Reply:
For some reason, I thought it would be added to the solution like some of the other packages. 😄

I'll remove it from there.